templates/zerver/help/change-stream-color.md

```diff
@@ -0,0 +1,38 @@
+# Change Stream Color
+Zulip allows you to change stream colors to better customize an environment
+that best suites you.
+
+### Using the streams sidebar
+The streams sidebar is the fastest approach to changing the color of a stream, to use this:
+
+1. Hover your cursor over the stream you would like to edit, then click the **down chevron**(![Down chevron](/static/images/help/chevron.png)) that will appear to the right of the stream name.
+2. From the pop-up menu select your desired stream color from the preset colors or open the color picker by clicking **Choose custom color**.
+
+![Change Stream Color](/static/images/help/stream-left-sidebar-color.png)
+
+You can also edit the color of a Stream from the **Subscriptions page**.
+
+### Finding the subscriptions page
+There are multiple ways to navigate to the subscriptions page. You can either:
+
+1. Click the **cog** (![cog](/static/images/help/cog.png)) icon in the top right corner.
+2. Select the **Manage Streams** option from the dropdown menu.
+
+![settings](/static/images/help/settings-dropdown-streams.png)
+
+or
+
+1. Click the **cog** (![cog](/static/images/help/cog.png)) icon located in the **left sidebar** next to **Streams**
+
+![Streams Cog](/static/images/help/streams-1.png)
+
+### Change Color
```

Comment:
A style comment: the capitalization of this heading doesn't match the others at the same level. Can you make the C in Color lower case?